AmplifyPlugins/Logging/Tests/AWSCloudWatchLoggingPluginTests/AWSCloudWatchLoggingSessionControllerTests.swift

```diff
@@ -67,6 +71,43 @@ final class AWSCloudWatchLoggingSessionControllerTests: XCTestCase {
         try await systemUnderTest.flushLogs()
         await fulfillment(of: [hubEventExpectation], timeout: 2)
     }
+    
+    /// Given: an AWSCloudWatchLoggingSessionController
+    /// When: the user identifier is changed
+    /// Then: a flush log is called and fails with a flushLogFailure hub event
+    func testResetLogsWhenIdentifierChanges() async throws {
+        let hubEventExpectation = expectation(description: "should receive flush failure event")
+        unsubscribeToken = Amplify.Hub.listen(to: .logging) { payload in
+            switch payload.eventName {
+            case HubPayload.EventName.Logging.flushLogFailure:
```

Comment:
is this expected? `flushLogs` failing when `userIdentifier` changes?

Reply:
that's outdated, can you refresh and see latest updates?